src/notification/telegram.ts

```diff
@@ -15,13 +15,18 @@ export function sendTelegramMessage(link: Link, store: Store) {
 
 		(async () => {
 			const givenUrl = link.cartUrl ? link.cartUrl : link.url;
+			const results = [];
 
-			try {
-				await client.sendMessage(telegram.chatId, `${Print.inStock(link, store)}\n${givenUrl}`);
-				logger.info('✔ telegram message sent');
-			} catch (error) {
-				logger.error('✖ couldn\'t send telegram message', error);
+			for (const chatId of telegram.chatId) {
+				try {
+					results.push(client.sendMessage(chatId, `${Print.inStock(link, store)}\n${givenUrl}`));
```

Comment:
we should make this `async` as well. But that's OK, we can do that in another PR.

Reply:
line 16 - 30 is already wrapped in an `async`, do you mean the body of the for loop should be `async` aswell?

Reply:
Oh, I mean that we should wrap with `Promise` or something.

Kind of like this:

https://github.com/jef/nvidia-snatcher/blob/8098a310925edb5c8738b00259856a2d64e2fec7/src/notification/discord.ts#L30-L35

But I don't even know if `sendMessage` returns a `Promise`, so I really don't know if this would work.